libbeat/docs/command-reference.asciidoc

```diff
@@ -0,0 +1,583 @@
+option is useful only for testing {beatname_uc}.
+
+endif::[]
+
+*`--memprofile FILE`*::
+Writes memory profile data to the specified output file. This option is useful
+for troubleshooting {beatname_uc}.
+
+ifeval::["{beatname_lc}"=="filebeat"]
+
+//REVIEWERS: I've written this topic to be consitent to what I see when I run -help, but I'm wondering why we aren't also showing the options that begin with a dash. For example, in the modules doc we show: -modules=nginx,mysql,system. Is --modules preferred?
+
+*`--modules MODULE_LIST`*::
+Specifies a comma-separate list of modules to run. For example:
++
+["source","sh",subs="attributes"]
+-----
+{beatname_lc} run --modules nginx,mysql,system
+-----
++
+Rather than specifying the list of modules every time you run {beatname_uc},
+you can use the <<modules-command,`modules`>> command to enable and disable
+specific modules. Then when you run {beatname_uc}, it will run any modules
+that are enabled.
+
+//REVIEWERS: Is the above syntax correct for --modules? We used to say -modules=nginx,mysql,system. Not sure if it makes a difference, but I'm trying to be consistent with what users see when they view --help.
+
+endif::[]
+
+ifeval::["{beatname_lc}"=="filebeat"]
+
+*`--once`*::
+When the `--once` flag is used, {beatname_uc} starts all configured harvesters
+and prospectors, and runs each prospector until the harvesters are closed. If
+you set the `--once` flag, you should also set `close_eof` so the harvester is
+closed when the end of the file is reached. By default harvesters are closed
+after `close_inactive` is reached.
+
+
+//REVIEWERS: Is the behavior of the harvesters/prospectors described above still true?
+
+endif::[]
+
+//REVIEWERS: I'm not documenting --setup because it will be removed in 6.0. Are there any other flags that I need to remove? Or mention that they're being maintained for backwards compability?
+
+ifeval::["{beatname_lc}"=="metricbeat"]
+
+*`--system.hostfs MOUNT_POINT`*::
+
+Specifies the mount point of the host's filesystem for use in monitoring a host
+from within a container.
+
+endif::[]
+
+ifeval::["{beatname_lc}"=="packetbeat"]
+
+*`-t`*::
+Reads packets from the pcap file as fast as possible without sleeping. Use this
+option in combination with the `-I` option. The `-t` option is useful only for
+testing Packetbeat.
+
+endif::[]
+
+{global-flags}
+
+*EXAMPLE*
+
+["source","sh",subs="attributes"]
+-----
+{beatname_lc} run -e
+-----
+
+
+[[setup-command]]
+==== `setup` command
+
+{setup-command-short-desc}.
+
+* The index template ensures that fields are mapped correctly in Elasticsearch.
+* The Kibana dashboards make it easier for you to visualize {beatname_uc} data
+in Kibana.
+* The machine learning jobs contain the configuration information and metadata
+necessary to analyze data for anomalies.
+
+//REVIEWERS: What are the requirements/permissions for users who want to load machine learning jobs? We need to document that here.
```

Comment:
@tsg I'm going to remove this comment from my review because it sounds like you'll be working on more docs related to ML and can update this topic if required.